### PR TITLE
QA-520: Increase test output verbosity

### PR DESCRIPTION
**Pull request by alfrunes**
The logs from test_demo_artifact failures contains insufficient information to explain why it is failing. The demo script exits with exit code 1 but all the containers are stopped correctly.

### CODE REVIEW

**Comment by alfrunes** [comment]
@mender-test-bot start pipeline

File: tests/tests/test_demo_artifact.py [hunk 1]
```diff
@@ -141,10 +140,18 @@ def demo_artifact_upload(self, run_demo_script, exit_cond="Login password:"):
...
+
         # Bring down the demo script
+        logger.info("-- Terminating demo script")
         proc.send_signal(signal.SIGTERM)
+        # Continue logging stdout until process stops
+        for line in proc.stdout:
+            logger.info(line.decode())
         proc.wait()
-        assert proc.returncode == 0
+        stderr = proc.stderr.read().decode()
```
Comment (lluiscampos) [inline tests/tests/test_demo_artifact.py:151]: `stderr` is not opened [here](https://github.com/mendersoftware/integration/blob/f899a6bcd2b3d9d4a0a345acb6e2e3bafd4f634f/tests/tests/test_demo_artifact.py#L77).

**Review by lluiscampos — APPROVED** [review]
Good addition :+1:

File: tests/tests/test_demo_artifact.py (hunk 1 above)
Comment (alfrunes) [reply tests/tests/test_demo_artifact.py:151]: Right, but setting this is tricky. What if it fills the OS pipe buffer and keeps the subprocess in a blocked state?

**Comment by alfrunes** [comment]
@mender-test-bot start pipeline

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**